src/R/Components/Impl/Resources.resx

```diff
@@ -304,7 +304,7 @@
     <value>R host process is not available.</value>
   </data>
   <data name="PackageManager_TransportError" xml:space="preserve">
-    <value>Connection to R host process failed.</value>
+    <value>Workspace is disconnected or R session is not running.</value>
```

Comment:
To fix #3217, I've split this message into three (individual for every tab), so you may skip this change.
Up to you.